install/terraform/modules/aks/aks.tf

```diff
@@ -45,33 +53,58 @@ resource "azurerm_resource_group" "test" {
 }
 
 resource "azurerm_kubernetes_cluster" "test" {
-  name                = "${var.cluster_name}"
-  location            = "${azurerm_resource_group.test.location}"
-  resource_group_name = "${azurerm_resource_group.test.name}"
+  name                = var.cluster_name
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
   dns_prefix          = "agones"
 
   kubernetes_version = "1.14.8"
 
   default_node_pool {
     name                = "default"
-    node_count          = 2
-    vm_size             = "${var.machine_type}"
-    os_disk_size_gb     = 30
+    node_count          = var.node_count
+    vm_size             = var.machine_type
+    os_disk_size_gb     = var.disk_size
     enable_auto_scaling = false
+    #enable_node_public_ip = true
+    #vnet_subnet_id     = azurerm_subnet.aks.id
   }
+
   service_principal {
-    client_id     = "${var.client_id}"
-    client_secret = "${var.client_secret}"
+    client_id     = var.client_id
+    client_secret = var.client_secret
   }
   tags = {
     Environment = "Production"
   }
 }
 
+resource "azurerm_kubernetes_cluster_node_pool" "system" {
+  name                  = "system"
+  kubernetes_cluster_id = azurerm_kubernetes_cluster.test.id
+  vm_size               = var.machine_type
+  node_count            = 1
+  os_disk_size_gb       = var.disk_size
+  enable_auto_scaling   = false
+  node_taints           = ["agones.dev/agones-system=true:NoExecute"]
+}
+
+/*
```

Comment:
Nit: remove commented out code.

Reply:
Sorry, this code was added due to my quota (of 4 VMs) limit has reached.
Tested with:
```
 terraform apply -var client_id=cb... -var client_secret=9e... -var node_count=1 --auto-approve
```
All is fine with this Metrics block also.